src/Network/Network/VirtualNetwork/Subnet/GetAzureVirtualNetworkSubnetConfigCommand.cs

```diff
@@ -17,44 +17,68 @@
 using System.Linq;
 using System.Management.Automation;
 using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
+using Microsoft.Azure.Commands.Network.Common;
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualNetworkSubnetConfig"), OutputType(typeof(PSSubnet))]
-    public class GetAzureVirtualNetworkSubnetConfigCommand : NetworkBaseCmdlet
-    {
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualNetworkSubnetConfig",DefaultParameterSetName = "GetByVirtualNetwork"), OutputType(typeof(PSSubnet))]
+    public class GetAzureVirtualNetworkSubnetConfigCommand : VirtualNetworkBaseCmdlet
```

Comment:
@lwajswaj what do we get by inheriting from `VirtualNetworkBaseCmdlet` rather than `NetworkBaseCmdlet`?

Reply:
I needed to change it so I can consume the method "GetVirtualNetwork" (line 58) since, otherwise, I would need to instance a full class and see how to do a cross-call to the other cmdlet. Open if there's a different way to do this.